src/components/RunningEmptyContent.vue

```diff
@@ -111,6 +133,17 @@ export default {
 	},
 
 	mounted() {
+		this.timer = setInterval(() => {
```

Comment:
Could you add a short explanation comment here to justify why it was not possible to use reactivity?